frontend/src/pages/auth/LoginSuccessPage.tsx

```diff
@@ -4,10 +4,7 @@ import { REACT_APP_REACT_SERVER_ORIGIN } from "../../shared/constants";
 export default function LoginSuccessPage() {
   useEffect(() => {
     if (window.opener) {
-      window.opener.postMessage(
-        "success",
-        `${REACT_APP_REACT_SERVER_ORIGIN}/patients`
-      );
```

Comment:
This file renders the popup page that users see after successfully logging in.

Here, I'm removing `/patients` so that the popup can communicate with both the Patients page and the Cohorts page. Only the website's origin is required by `window.opener.postMessage()`.